notebooks/01_prepare_data/wikidata_knowledge_graph.ipynb

```diff
@@ -566,9 +561,9 @@
  "metadata": {
   "celltoolbar": "Tags",
   "kernelspec": {
-   "display_name": "Python (reco_base)",
+   "display_name": "Python (reco_bare)",
    "language": "python",
-   "name": "reco_base"
+   "name": "reco_bare"
```

Comment:
I guess you've been updating an ancient version of our environment :-) instead of re-installing

Reply:
good catch, changed